### ISSUE TITLE
No Search Results & failure to load information. 

### ISSUE THREAD

**Issue by vhon10** [mylar3/mylar3#1039]
**Describe the bug**
Hi, really struggling to get this thing working correctly. I've downloaded and set-up as per the instructions. 

-I've got the CV API Key set up and enabled, no trouble. 
-I've got a search provider set (1337x through Prowlarr). 
-I've got a download client set (qbit).
-I've set a comic location path. 

I can see the "This week" list and even add comics as wanted through that list, however, it then fails to load the comic information, cover etc.

The search returns zero results regardless of search terms used, it simply states "Processing".  

I recognize that there is almost certainly an error on my part thats breaking something, any help appreciated. 

**CarePackage**

[carepackage.zip](https://github.com/mylar3/mylar3/files/7982241/carepackage.zip)

**Environment (please complete the following information):**
 - OS: WIdnows 10
 - Branch / commit: Latest version (0.6.3). 
 - Using Python version 3.8.1 with the py launcher. 

**Additional information**
Also tried running mylar.py from an elevated CMD incase its a permission issue, no dice.

**Comment by evilhero** [comment]
Probably a bunch of things, but kind of hard to make sense of with the logs being redacted.

Your qBit host:ip address in Mylar - you need to precede the host with the protocol (``http://`` or ``https://``)

It looks like you're having problems connecting to CV for whatever reason. 

Can you ping or get to ComicVine's website on the machine running Mylar - even thru a web browser ? If you can - you can check your API hits on their api site to see if you're going over the hourly limit, especially if you're using the same CV key across different programs.

If you open up your config.ini, make sure that the ``comicvine_api`` field is not within quotation marks (which would mean that there's a blank space in there).

Other than that, your best bet might be to hop on discord so we can try to help out better, if none of the above help.

**Comment by vhon10** [comment]
Qbit did have the http prefix on it, but in fairness I don't think I restarted before creating the care-package so perhaps it hadn't been applied. 

Definitely no-where near hitting the Comic vine API cap as this is my first time messing around with it (If that wasn't obvious). Site confirms no rate current limits.

The comicvine_api field in the config was not in quotations. However, as I was checking I did notice that it had the unusal prefix of "none". Which means I managed to bugger up a simple copy paste. Fixing that resolved the issue. (Although I swear Mylar accepted the bad key, the warning disappeared anyway?). 

I've been banging my head against a wall for a couple hours over literally nothing. Thanks again for the support.
